### PR TITLE
AArch64: Set flushing mode for subnormals on glibc and bionic based systems

### PR DESCRIPTION
**Pull request by pawosm-arm**

### CODE REVIEW

File: lib/evaluate/host.cc [hunk 1]
```diff
@@ -42,6 +42,24 @@ void HostFloatingPointEnvironment::SetUpHostFloatingPointEnvironment(
...
+    currentFenv_.__fpcr &= ~(1U << 24);  // control register
+  }
+#elif defined(__BIONIC__)
+  if (context.flushSubnormalsToZero()) {
+    currentFenv_.__control |= (1U << 24);  // control register
+  } else {
+    currentFenv_.__control &= ~(1U << 24);  // control register
+  }
+#else
+  // TODO other C libraries on AArch64
```
Comment (klausler) [inline lib/evaluate/host.cc]: Why not have an `#error` or `static_assert` here?  Emitting an error message at the user doesn't seem right.  And they'll get spammed with a message every time this gets called.

Comment (pawosm-arm) [reply lib/evaluate/host.cc]: It's just to keep consistency with the code immediately below.

Comment (klausler) [reply lib/evaluate/host.cc]: Somebody's going to have to fix that code immediately below someday, I think, and now they'll have to fix it twice.

Comment (jeanPerier) [reply lib/evaluate/host.cc]: @tskeith had already suggested `#warning`. I had not applied it. We should also change the other `#else` case then.
However, we should also white-list  host architectures that cannot control the option (e.g power (at least power 8), I am not sure for power9).
Then we should have a user warning if the requested subnormal flushing cannot be obtained on the host.
@Paul can you make both messages `#error` or static_assert ? I will handle the power case when adding the test case.

Comment (jeanPerier) [reply lib/evaluate/host.cc]: @klausler, @pawosm-arm sorry for the confusion, I can fix all warnings in my branch PR.

Comment (pawosm-arm) [reply lib/evaluate/host.cc]: @jeanPerier so do I still need to make changes?

Comment (jeanPerier) [reply lib/evaluate/host.cc]: No, I will take care of changing both warning messages to compile time errors while taking care of Power and adding the test case.

**Pull request merged**